### ISSUE TITLE
Create dockerfile with all pre-requites to run on Azure Kubernetes cluster

### ISSUE THREAD

**Issue by Soham05** [google/android-emulator-container-scripts#129]
Hi, I am trying to deploy the docker image on Azure Kubernetes cluster, however it requires all prerequisites to be a part of the docker image. Is it possible to add all these installations as a part of dockerfile used to generate the current image (from emu_docker.py) ? if yes, how can I do the same?

**Comment by pokowaka** [comment]
Yes, you can edit the [dockerfile](https://github.com/google/android-emulator-container-scripts/blob/master/emu/templates/Dockerfile) inside the template directory. 

Feel free to share your changes, someway might be able to make it available to others.

**Comment by iremmats** [comment]
Doing the same. Did you get this to work? @Soham05

**Comment by jpcottin** [comment]
The container image already includes everything the emulator needs at runtime — the AKS-specific work is on the cluster side (KVM / `/dev/kvm` exposure to the pod) rather than baked into the Dockerfile. Same scope as #281. Closing as out-of-scope for the image build.
